docs/2.0/docs/pipelines/guides/updating-plan-apply-roles.md

```
@@ -0,0 +1,500 @@
+
+## Identifying existing roles
+
+<Tabs groupId="platform">
+<TabItem value="github" label="GitHub" default>
+
+<Tabs groupId="cloud">
+<TabItem value="aws" label="AWS" default>
+
+Your plan and apply roles are managed by the bootstrap stack in your Terragrunt Scale repository. Replace <CustomizableValue id="ACCOUNT_NAME" /> with the name of the account you are working with. This stack is initially created at:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)